### DISCUSSION TITLE
Macro-length limited to 5

### DISCUSSION THREAD

**Discussion by Sentient-Toaster** [kriomant/ch57x-keyboard-tool#73] [Q&A]
Sorry if this is in the wrong spot here.

I was wondering whether the macro-length per button being limited to 5 is hardware or software dependend, if that is known?

Currently having more than 5 keystrokes in the mapping.yaml will validate, but the keyboard will only execute the first 5.

(Edit: Using the program from the vendor I can upload macros with a length >5, so it probably isn't hardware-limited, at least with the device I have. Is there any way to introduce this functionality to this software?)

Thanks in advance for any help, really thankful to have found this program!

**Comment by lfcorullon** [comment]
Hey, there... anyone knows how to remove this limitation of 5 characters???

**Reply by kriomant** [reply]
Assign some shortcut to keypad key and use external automation software to map this shortcut to any number of keys.

**Comment by lfcorullon** [comment]
I want the keypad to automate windows login. This is not an option. If I
can remove the limitation of 5 characters it'll be awesome.


______________
Luis Felipe Corullón
***@***.***
(51) 99685.7565

Em dom., 29 de set. de 2024, 21:44, Mikhail Trishchenkov <
***@***.***> escreveu:
